### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by MaureenElsberry**
Adds the Arrow code of conduct and contact information.

### CODE REVIEW

**Review by rachelcarmena — APPROVED** [review]
Thank you so much @MaureenElsberry !! :clap: :tada:

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
```
Comment (nomisRev) [inline CODE_OF_CONDUCT.md:130]: I'm not sure what `homepage` refers to here. Should these be links instead, `[Contributor Covenant](https://www.contributor-covenant.org)`?

**Review by nomisRev — APPROVED** [review]
Looks great @MaureenElsberry! Thank you so much!!
Only a small question about the links at bottom of COC.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rachelcarmena) [reply CODE_OF_CONDUCT.md:130]: Good point @nomisRev ! :raised_hands:  It comes from the template:

Screenshot

However, it's not shown when rendering the page:

Screenshot

Let's merge it and I'm going to try to communicate the status of the template to the creators.

**Pull request merged**